File: JavaScript/Sorting/SelectionSort.js
function SelectionSort(l){

    let start, minpos, i, temp;
    for (start = 0; start < l.length; start++){

        minpos = start
        for(i = start; i<l.length; i++){
            if(l[i] < l[minpos]){
                minpos = i
            }
        }
        temp = l[start];
        l[start] = l[minpos];
        l[minpos] = temp
    }
    return l
}

let list1 = [9,2,3,7,5,8,4,6]
console.log(SelectionSort(list1))

File: Python/Sorting/SelectionSort.py
def SelectionSort(l):
    for i in range(len(l)):
        min_val = i

        for j in range(i+1, len(l)):
            if l[j] < l[min_val]:
                min_val = j

        (l[i], l[min_val]) = (l[min_val], l[i])

    return l

list1 = [9,2,4,7,3,8,5]
print(SelectionSort(list1))